frontend/src/metabase/core/components/Toggle/Toggle.styled.tsx

```diff
@@ -41,6 +41,7 @@ export const ToggleRoot = styled.input<ToggleRootProps>`
   background-color: ${color("bg-medium")};
   background-color: ${getBackgroundColor};
   transition: background-color 0.3s;
+  flex-shrink: 0;
```

Comment:
Fix toggle being shrinked in flex containers

#### Before
![image](https://user-images.githubusercontent.com/1937582/175020892-762362f6-82a9-4325-b8fc-92f977df4cbe.png)

#### After
![image](https://user-images.githubusercontent.com/1937582/175020957-fc9cc059-58bd-458f-8dd1-d07bab366b11.png)